addons/web_editor/static/src/js/editor/image_processing.js

```diff
@@ -437,7 +437,8 @@ async function loadImageInfo(img, rpc, attachmentSrc = '') {
         params: {src: src.split(/[?#]/)[0]},
     });
     // Check that url is local.
-    const isLocal = original && new URL(original.image_src, window.location.origin).origin === window.location.origin;
+    const isLocal = original && new URL(original.image_src, window.location.origin).origin === window.location.origin 
+        && !/\/web\/image\/\d+-redirect\//.test(original.image_src);
```

Comment:
I'd have added a comment here because I don't think people reading this will know about the fact it's a redirect image attachment due to external URL uploaded.